parcel/ParcelTendencies.jl

```diff
@@ -223,6 +229,40 @@ function condensation(params::CondParams, PSD_liq, state, ρ_air)
     return qₗ + dqₗ_dt * const_dt > 0 ? dqₗ_dt : -qₗ / const_dt
 end
 
+function condensation(params::NonEqCondParams_Anna, PSD, state, ρ_air)
+
+    FT = eltype(state)
+    (; Sₗ, T, qₗ, qᵥ, qᵢ) = state
+    (; tps, liquid) = params
+
+    q_sat_liq = max(Sₗ * qᵥ - qᵥ, 0)
+    q_sat = TD.PhasePartition(FT(0), q_sat_liq, FT(0))
+
+    q = TD.PhasePartition(qᵥ + qₗ + qᵢ, qₗ, qᵢ)
+
+    new_q = MNE.conv_q_vap_to_q_liq_ice(liquid, q_sat, q)
+
+    return new_q
+end
+
+function condensation(params::NonEqCondParams, PSD, state, ρ_air)
+    FT = eltype(state)
+    (; T, qₗ, qᵥ, qᵢ) = state
+
+    (; tps, liquid, dt) = params
+
+    q = TD.PhasePartition(qᵥ + qₗ + qᵢ, qₗ, qᵢ)
+
+    cond_rate = MNE.conv_q_vap_to_q_liq_ice_MM2015(liquid, tps, q, ρ_air, T)
+
+    # using same limiter as ClimaAtmos for now
```

Comment:
Would be good to do some testing and see if we need the limiter. Maybe we don't? Or if we do, maybe we can limit just by dt?